api-handler/dataAccess.js

```diff
@@ -5,12 +5,16 @@ const cul = require("./getAllCultures");
 const hou = require("./getAllHouses");
 const reg = require("./getAllRegions");
 const tit = require("./getAllTitles");
+const pop = require("./getPopularity");
+const rel = require("./support/deadRelations");
 
 const url_cha = 'https://got-api.bruck.me/api/characters/';
```

Comment:
populartity is currently gotten from the file u provided and the relation to dead characters is getting scrapped from the wiki so there are no api calls being made at the moment